e2e/07-btc-payments-receive-flow.e2e.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { i18nObject } from "../app/i18n/i18n-util"
+import { loadLocale } from "../app/i18n/i18n-util.sync"
+import { goBack, selector } from "./utils"
+import { payInvoice } from "./utils/graphql"
+
+describe("Btc Receive Payment Flow", async () => {
+  loadLocale("en")
+  const LL = i18nObject("en")
+  const timeout = 30000
+  let invoice: string
+  let paymentStatus: string | null | undefined
+
+  it("Click Receive", async () => {
+    const receiveButton = await $(selector(LL.MoveMoneyScreen.receive(), "Other"))
+    await receiveButton.waitForDisplayed({ timeout })
+    await receiveButton.click()
+  })
+
+  it("Click Copy Invoice", async () => {
+    let copyInvoiceButton
+    if (process.env.E2E_DEVICE === "ios") {
+      copyInvoiceButton = await $('(//XCUIElementTypeOther[@name="Copy Invoice"])[2]')
+    } else {
+      copyInvoiceButton = await $(selector("Copy Invoice", "Button"))
+    }
+    await copyInvoiceButton.waitForDisplayed({ timeout })
+    await copyInvoiceButton.click()
+  })
+
+  it("Get Invoice from clipboard (android) or share link (ios)", async () => {
+    await browser.pause(2000)
+    if (process.env.E2E_DEVICE === "ios") {
+      // on ios, get invoice from share link because copy does not
+      // work on physical device for security reasons
+      const shareButton = await $('(//XCUIElementTypeOther[@name="Share Invoice"])[2]')
+      await shareButton.waitForDisplayed({ timeout: 8000 })
+      await shareButton.click()
+      const invoiceSharedScreen = await $('//*[contains(@name,"lntbs")]')
+      await invoiceSharedScreen.waitForDisplayed({
+        timeout: 8000,
```

Comment:
how long is the default timeout? is the default not ok?

Reply:
default is `30000`. We don't need to wait that long for this

Reply:
but what is the point of setting de default value? it will automatically move on if it finds the element sooner?